src/pages/collections/_itemId/index.vue

```diff
@@ -0,0 +1,298 @@
+
+export default Vue.extend({
+  mixins: [imageHelper, formsHelper, itemHelper],
+  validate(ctx: Context) {
+    return isValidMD5(ctx.route.params.itemId);
+  },
+  async asyncData({ params, $userLibrary, store }) {
+    const itemId = params.itemId;
+
+    if (!store.getters['items/getItem'](itemId)) {
+      await $userLibrary.getItem(itemId);
+    }
+
+    return { itemId };
+  },
+  data() {
+    return {
+      itemId: '' as string,
+      parentItem: {} as BaseItemDto,
+      currentTab: 0,
+      loading: true,
+      viewType: '',
+      backdropImageSource: '',
+      currentVideoTrack: undefined as number | undefined,
+      currentAudioTrack: undefined as number | undefined,
+      currentSubtitleTrack: undefined as number | undefined
+    };
+  },
+  async fetch() {
+    this.loading = true;
+    await this.getCollectionChildren({ itemId: this.item.Id }).finally(
+      () => (this.loading = false)
+    );
+  },
+  head() {
+    return {
+      title: this.title
+    };
+  },
+  computed: {
+    ...mapGetters('items', ['getItem']),
+    ...mapGetters('collections', ['getChildren']),
+    ...mapState('page', ['title']),
+    item(): BaseItemDto {
+      return this.getItem(this.itemId);
+    },
+    children(): BaseItemDto[] {
+      return this.getChildren(this.item.Id);
+    },
+    crew(): BaseItemPerson[] {
+      let crew: BaseItemPerson[] = [];
+
+      if (this.item.People) {
+        crew = this.item.People.filter((person: BaseItemPerson) => {
+          return ['Director', 'Writer'].includes(person.Type || '');
+        });
+      }
+
+      return crew;
+    },
+    actors: {
+      get(): BaseItemPerson[] {
+        if (this.item.People) {
+          return this.item.People.filter((person: BaseItemPerson) => {
+            return person.Type === 'Actor';
+          }).slice(0, 10);
+        } else {
+          return [];
+        }
+      }
+    },
+    directors: {
+      get(): BaseItemPerson[] {
+        return this.crew.filter(
+          (person: BaseItemPerson) => person.Type === 'Director'
+        );
+      }
+    },
+    writers: {
+      get(): BaseItemPerson[] {
+        return this.crew.filter(
+          (person: BaseItemPerson) => person.Type === 'Writer'
+        );
+      }
+    }
+  },
+  watch: {
+    item: {
+      handler(val: BaseItemDto): void {
+        this.setPageTitle({ title: val.Name });
+
+        const hash = this.getBlurhash(val, ImageType.Backdrop);
+
+        this.setBackdrop({ hash });
+      },
+      immediate: true,
+      deep: true
+    }
+  },
+  mounted() {
+    this.setAppBarOpacity({ opaqueAppBar: false });
+  },
+  destroyed() {
+    this.setAppBarOpacity({ opaqueAppBar: true });
+    this.clearBackdrop();
+  },
+  methods: {
+    ...mapActions('page', ['setPageTitle', 'setAppBarOpacity']),
+    ...mapActions('backdrop', ['setBackdrop', 'clearBackdrop']),
+    ...mapActions('collections', {
+      getCollectionChildren: 'getCollectionChildren'
```

Comment:
This is only needed if we want to change the way that function is invoked inside this vue instance. As the name you used is the same, you can simply map it as the other stores.